### ISSUE TITLE
How to install on raspberry Pi

### ISSUE THREAD

**Issue by Cammate** [jcalado/cgtimer#5]
Hi,
I would like to run your timer on a raspberry Pi but don't know the steps to take to make it run. There is no install-guide in the readme so I am a bit lost here.
Can you help me with this?
Kind regards,
Cammate

**Comment by jcalado** [comment]
Hi!

So, to build from source you should run, from the source code folder:

```
npm install
npm run dev
```

**Comment by Cammate** [comment]
Hi Joel,
Thanks for your quick response. I managed to get a bit further, but I get a lot of errors and deprecated warnings. I installed NodeJS and Electron, updated everything. I think I have to run "node index.js" on the main-folder in the src-folder, but that gives me an syntax-error. So after several hours of searching for solutions it isn't running at all.
I guess my knowledge is not deep enough to get this to work. :(
Again, thanks for your help. I think I have to find another simpler to implement solution to show both Time of Day and the remaining of the cliplength out of Caspar. But don't know if there is any. If you have any other suggestion, I am open for it.
Kind regards,
Cammate
